keep-ui/features/workflow-execution-results/ui/WorkflowExecutionResults.tsx

```diff
@@ -232,16 +238,35 @@ export function WorkflowExecutionResultsInternal({
       )}
       <ResizableColumns initialLeftWidth={50}>
         <div className="pr-2">
-          <Card className="p-0 overflow-hidden">
-            <WorkflowExecutionLogs
-              logs={logs ?? null}
-              results={results ?? null}
-              status={status ?? ""}
-              checks={checks}
-              hoveredStep={hoveredStep}
-              selectedStep={selectedStep}
-            />
-          </Card>
+          {logs ? (
+            <Card className="p-0 overflow-hidden">
+              <WorkflowExecutionLogs
+                logs={logs ?? null}
+                results={results ?? null}
+                status={status ?? ""}
+                checks={checks}
+                hoveredStep={hoveredStep}
+                selectedStep={selectedStep}
+                isLoading={isLoading}
+              />
+            </Card>
+          ) : (
+            <EmptyStateCard
+              title="No logs found"
+              description="The workflow is still running"
```

Comment:
Won't this be confusing if there are no logs on workflow failure?

Reply:
On works failure there should be failure logs, like on the video in the PR description.